releases/release-announcement-guide.md

```diff
@@ -0,0 +1,306 @@
+
+
+# Blog Post
+
+
+## Blog Overview
+
+
+
+* Publish a blog of 1,000 words or less that reviews the highlights of new release
+* The blog will promote themes and the release’s top features.  The blog is a summary of the release, and more detailed information is provided in links to GitHub Issues, Pull Requests (PR), additional (Working Group or User) blogs, repos or other resources.
+* The post uses the GitHub PR fast pages process, which documented here, ​​https://github.com/kubeflow/blog#blog-submission-instructions
+    * This PR is an example of the 1.4 blog post, [https://github.com/kubeflow/blog/pull/109](https://github.com/kubeflow/blog/pull/109) 
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the blog’s creation and release process,  creates initial version, asks and incorporates edits, keeps process moving, provides updates to Community
+* GitHub specialist - 10 hours
+    * Creates PR, helps to check content, incorporate edits to PR
+* Content creators, editors and approvers - 2 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Final Approver - 4 hours
+    * Kubeflow Steering Committee lead
+
+
+## Example 
+
+An example of a release blog post is here: https://blog.kubeflow.org/kubeflow-1.4-release/
+
+Note - A sample of the work activities and timeline are detailed in a subsequent section of this document.
+
+
+# Presentation
+
+
+## Presentation overview
+
+30 minute slide presentation recorded and presented by the Kubeflow Community Product Management, the Release Team and the Working Group Leads.   The presentation should be posted on the [Kubeflow Youtube channel](https://www.youtube.com/channel/UCNSJwHkt2f_cP_UN2q-yJlw).
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the process, creates initial presentation outline, asks working groups for content and speakers, reviews and incorporates edits, keeps, provides updates to Community, keeps the process moving
+* Content creators, editors and approvers - 3 hours
+    * Working Group leads provide content, review text, approve via /lgtm in PR
+* Video Editor
+    * We need a pro at editing videos (that are currently created via Zoom)
+* Final Approver - 2 hours
+    * Kubeflow Steering Committee lead
+
+
+## Example: 
+
+An example of the recorded presentation is here: [https://www.youtube.com/watch?v=gG61gHw4J14](https://www.youtube.com/watch?v=gG61gHw4J14)
+
+An example of the slide is available here: [https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651](https://docs.google.com/presentation/d/1S7FRa8DI5uymUuPv56ioC37QXgb7kxSC5p7pp0gMd9U/edit#slide=id.gba45ef4eaa_0_1651)
+
+
+# Sample Activity Timeline
+
+This timeline shows potential activities for the blog and presentation, starting from 35 days from the postings.
+
+
+<table>
```

Comment:
I found this table really hard to follow especially with the time being part of the content. Would it make sense to have a separate column for time? 

Also, while I can follow the requirements, I am not sure who is completing these actions listed. Could we add who is responsible for the tasks? Something similar to below?

| Time | Who | Blog | Presentation |
| -- | -- | --| -- |
| 35 days before the release | Product Manager | action for blog post | action for presentation |